docs/azure/intro.md

```diff
@@ -23,6 +23,8 @@ This document provides an overview of key concepts and services .NET developers
 
 **Managed services**: Azure provides some services where you provide data or information to Azure, and Azure's implementation takes the appropriate action. One example is Azure Blob Storage, where you provide files and Azure handles reading, writing, and persisting them.
 
+**Azure SDK for .NET**: Sometimes referred to as the **Azure libraries for .NET**, this collectively refers to the [NuGet packages](https://www.nuget.org/profiles/azure-sdk) you can install in your project to provide connectivity to various Azure resources. These packages also include management libraries used to provision and administer the resources.
+
```

Comment:
@IEvangelist What do you think about adding this definition here?

Reply:
I made a suggestion